### ISSUE TITLE
Try converting these textures

### ISSUE THREAD

**Issue by MARK2580** [matyalatte/UE4-DDS-Tools#30]
Hello. Here are some textures from the game The day before. Engine version UE5 5.2.1.0
I was never able to get anything adequate from these files, even through rawtex / texconv
[thedaybefore_assets.zip](https://github.com/matyalatte/UE4-DDS-Tools/files/15002629/thedaybefore_assets.zip)

**Comment by matyalatte** [comment]
Hi, thanks for the report.
Do you have more infomation about your assets?

`T_drum_kit_01_d.uasset` says it's a 2d texture. and `T_drum_kit_01_d.uexp` says it's a 1024x1024 texture that have DXT1 format. Its DDS data should be 683KB at most but `T_drum_kit_01_d.ubulk` is 830KB. (Usually, the texture ubulk has only pixel data for DDS.)
Also, I can see `PF_DXT1` twice in `T_drum_kit_01_d.uexp`. I've never seen like this before.
I assume that the day before uses a custom version of UE or a less well-known option.

**Comment by MARK2580** [comment]
no, unfortunately these files, the version of the game and what kind of game this is is the only thing I can give you, I’m very far from all this stuff with coding) if you need more examples... by the way, the game also contains ordinary textures that umodel was able to immediately convert to a readable format.

**Comment by matyalatte** [comment]
I confirmed that they are [streaming virtual textures](https://dev.epicgames.com/documentation/en-us/unreal-engine/streaming-virtual-texturing-in-unreal-engine?application_version=5.2
).
I got similar structures when I enabled the `Virtual Texture Streaming` option in Unreal Engine.
Presumably, their pixel data is subdivided into small pieces and stored in specific data structures.
But unfortunately, it's beyond my understanding. Idk how to convert them to the standard image formats.
So, I added an error message for virtual textures at 3d6ac1c2708fd9680d809740c628409dd904c717, and gave up to support your assets.

**Comment by MARK2580** [comment]
It’s a pity, I’m increasingly encountering these virtual textures, even though they were originally just an ordinary picture =(

**Comment by Mayki07** [comment]
I apologize in advance for a question that doesn't quite belong here, but could you give me some advice on how to properly set up the virtual texture generation in the unreal editor? I've exported the texture in png via Umodel, edited it in Photoshop and would like to create a uasset back from it directly in the editor. But I can't seem to set it up properly.

![UE](https://github.com/matyalatte/UE4-DDS-Tools/assets/35667524/4d925244-0fbc-4ee8-b25b-b96ccd775044)

I always end up with a black texture instead of a picture.

![Robo](https://github.com/matyalatte/UE4-DDS-Tools/assets/35667524/37574502-4528-4aae-9089-611127f9252c)

**Comment by MARK2580** [comment]
Quite by accident I discovered that Fmodel is now partially able to load virtual textures and that it uses this library, it may be useful...  https://github.com/FabianFG/CUE4Parse

These are the same drums whose texture I sent in the very first message.
![scr_16_07_2024_14_18_45](https://github.com/user-attachments/assets/0658b2e1-85fb-4ce8-ba89-dd840bf9429a)

**Comment by RenegadeWraith** [comment]
Really hope there will be an update. All future UE games will pretty much rely on virtual textures. 
Fmodel can view and export virtual textures but no way to modify...
